utils/common/src/main/java/brooklyn/util/javalang/Threads.java

```diff
@@ -44,7 +46,15 @@ public static boolean removeShutdownHook(Thread hook) {
             return Runtime.getRuntime().removeShutdownHook(hook);
         } catch (IllegalStateException e) {
             // probably shutdown in progress
-            log.debug("cannot remove shutdown hook "+hook+": "+e);
+            String text = Exceptions.collapseText(e);
+            if (text.contains("Shutdown in progress")) {
+                if (log.isTraceEnabled()) {
+                    log.trace("Could not remove shutdown hook "+hook+": "+text);
```

Comment:
Normally I'd agree but this is a known exception which is noise in the logs, only arising from one place.  And it appears at the very end of the log and so is very visible, giving the wrong impression if it looks like a real problem.